app/uk/gov/hmrc/rdsdatacacheproxy/controllers/DirectDebitController.scala

```diff
@@ -57,3 +62,17 @@ class DirectDebitController @Inject()(
     authorise.async(parse.json[CreateDirectDebitRequest]):
       implicit request =>
         Future.successful(Created(request.body.paymentReference))
+
+
+  def getWorkingDaysOffset(): Action[WorkingDaysOffsetRequest] =
+    authorise.async(parse.json[WorkingDaysOffsetRequest]):
+      implicit request =>
+        val maybeCurrentDate: Option[LocalDate] = Try(LocalDate.parse(request.body.baseDate)).toOption
+        val maybeNumberOfWorkingDays: Option[Int] = Try(request.body.offsetWorkingDays.toInt).toOption
```

Comment:
These lines aren't optional, the request will have already returned a 400 if these values aren't present because of L68.
This will remove the entire `match` statement too